iphone/Classes/TiUIiOSProxy.m

```diff
@@ -841,5 +841,16 @@ - (id)createWebViewProcessPool:(id)args
 MAKE_SYSTEM_PROP(INJECTION_TIME_DOCUMENT_END, WKUserScriptInjectionTimeAtDocumentEnd);
 #endif
 
+- (TiColor *)fetchSemanticColor:(id)color
+{
+  ENSURE_SINGLE_ARG(color, NSString);
+
+  if ([TiUtils isIOSVersionOrGreater:@"11.0"]) {
+    return [[TiColor alloc] initWithColor:[UIColor colorNamed:color] name:nil];
+  } else {
+    return [[TiColor alloc] initWithColor:UIColor.blackColor name:@"black"];
```

Comment:
While this is undocumented, it's still callable by a user. So I kept the fallback to black like in the darkmode plugin/module by Hans.